src/components/DefaultPage.js

```diff
@@ -1,27 +1,38 @@
-import React from 'react'
+import React, {useState} from 'react'
 import TypeIt from 'typeit-react'
 import '../index.css'
+import SignInModal from './SignInModal'
+import SignUpModal from './SignUpModal'
+
 
 const DefaultPage = () => {
-  return (
+  const [showSignInModal, setShowSignInModal]= useState(false)  
+  const [showSignUpModal, setShowSignUpModal]= useState(false)  
+return (
     <>
     <div className='default' >
         <nav className='default-nav' >
             <h2>letswatch</h2>
-            <button className='sign-in-btn' > Sign in</button>
+            <button className='sign-in-btn' onClick={() => {setShowSignInModal(true); setShowSignUpModal(false)}}> Sign in</button>
         </nav>
         <div className='welcomeMsg' >
-            <h1><TypeIt options={{ loop: true }}>Welcome to letswatch. </TypeIt>  </h1> 
+            <h1>
+              <TypeIt options={{ loop: true }}>
+                Welcome to letswatch. 
+              </TypeIt>
+            </h1> 
             <h2> Your one-stop-shop for cinematic entertainment! </h2> 
             <h3> So sit back, relax, and let us take you on a journey through the art of storytelling.</h3>
             <div className='welcomeBtn'>
-              <button className='get-started-btn' >Get Started Here</button>
+              <button className='get-started-btn' onClick={() => {setShowSignUpModal(true); setShowSignInModal(false)}}>Get Started Here</button>
```

Comment:
instead of calling 2 setter functions here, its better to delegate this call to just 1 function which you can call **`showLoginFormHandler`** or **`handleShowLoginForm`**... ("handle/handler" is kindof a conventions in the javascript community to show that a function is meant to handle an event).. then inside that function, you can call these 2 setter functions

Reply:
![Screenshot (137)](https://user-images.githubusercontent.com/73981407/221430854-662c905a-8fd5-4169-8a24-6aaecd17693f.png)